src/app/App.jsx

```diff
@@ -1,55 +1,43 @@
-import React, { Component } from "react";
+import React, { Component, useEffect, useState } from "react";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
-
 import { hasValidAuthToken } from "./utilities/hasValidAuthToken";
 import user from "./utilities/api-clients/user";
 import log from "./utilities/logging/log";
 import ping from "./utilities/api-clients/ping";
-
-import Notifications from "./global/notifications/Notifications";
+import Notifications from "./components/notifications";
 import notifications from "./utilities/notifications";
 
-const propTypes = {
-    children: PropTypes.node,
-    dispatch: PropTypes.func.isRequired,
-    notifications: PropTypes.arrayOf(PropTypes.object)
-};
-
-class App extends Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isCheckingAuthentication: false
-        };
-    }
+const App = props => {
+    const [isCheckingAuthentication, setIsCheckingAuthentication] = useState(true);
```

Comment:
The old code initially set isCheckingAuthentication to false but the new code is setting it too true. Is this correct?